solr/core/src/java/org/apache/solr/cloud/ReplicateFromLeader.java

```diff
@@ -74,14 +74,18 @@ public void startReplication(boolean switchTransactionLog) {
               "SolrCore not found:" + coreName + " in " + CloudUtil.getLoadedCoreNamesAsString(cc));
         }
       }
+
       SolrConfig.UpdateHandlerInfo uinfo = core.getSolrConfig().getUpdateHandlerInfo();
+      String customPollInterval =
+          core.getSolrConfig().get("updateHandler").get("commitPollInterval").txt();
```

Comment:
implementation detail: might a `UpdateHandlerInfo.commitPollInterval` member be an alternative?

Reply:
+1 on that, then I think in determinePollInterval you could just account for the configured commitPollInterval